Add documentation node on telemetry and benchmarking (#4)
cloudflare/odoh-client-go: README.md

```sh
./odoh-client odohconfig-fetch --target odoh.cloudflare-dns.com --pretty
```

### Note

> > The current implementation of the go lang client has a sub command for benchmarking various protocols and has been

Reviewer: ```suggestion
> This tool includes a sub command for benchmarking various protocols and has been
```
